py3status/modules/mpd_status.py

```diff
@@ -156,34 +186,31 @@ def attr_getter(attr):
                 text = self.py3.safe_format(self.format, attr_getter=attr_getter)
 
         except socket.error:
-            text = "Failed to connect to mpd!"
-            state = None
+            text = self.mpd_no_conn
         except CommandError:
-            text = "Failed to authenticate to mpd!"
-            state = None
+            text = self.mpd_no_auth
```

Comment:
I started off with `state_no_mpd` but it feels nice (fits in with other states, but not part of mpd, just outside) so I go with `mpd_`*. If mpd connection or mpd auth failed, it's generally bad... I modified code (to use RED for else). All tested.

One thing to mention. I change default for `hide_when_stopped` because Idk how to preserve that when I'm deprecating both options. I feel that it shouldn't go away when stopped as a default option... I tested this. All okay. If people want to hide them, they should have that option already... If not, they will see to use `state_stop = ""` instead.